.travis.yml

```diff
@@ -15,25 +15,25 @@ matrix:
     fast_finish: true
     include:
           # Minimum supported dependencies with the latest and oldest PHP version
-        - php: 7.2
+        - php: 7.3
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
         - php: 7.1
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
 
           # Test the latest stable release
         - php: 7.1
-        - php: 7.2
```

Comment:
We should not remove 7.2. Right?

Reply:
Right. I though we still got it from the matrix, but we don't. Fixing now.